packages/components/Button/src/Button.settings.win32.ts

```diff
@@ -81,6 +81,11 @@ export const settings: IComposeSettings<IButtonType> = [
           borderColor: 'strokeFocus2',
           borderWidth: 2,
         },
+        stack: {
+          style: {
+            height: 22
+          }
+        }
```

Comment:
Feels weird to shrink the button on focus. Is this how other win32 controls behave?

Reply:
No, I didn't see these issues with other controls. But without it the height of the button is changing even with fixed height

Reply:
I recall seeing this bug when I updated the border for buttons in a previous PR. I made a comment about it here: https://github.com/microsoft/fluentui-react-native/pull/954#discussion_r696976520

I think adding borders affects a property on the win32 platform that causes the border to thicken AND add space around the control. I'm not sure if this is desired or if it's a bug but I'll create a work item to investigate it. 

I suppose this workaround is alright for the time being but I don't think it should be a permanent solution. @ValentinaKozlova could you add a comment above the style describing the bug so we know that it's there and can remove the effect if it gets fixed natively?

Reply:
I see you already created a bug for it in ADO. Thanks @ValentinaKozlova!

Reply:
The designs are assuming that borders are drawn inside the control, not outside. So even though there are 2px borders for focus, in the redlines it doesn't change the height since those borders are "inset" and taking away from the internal space, if that makes sense.

Unfortunately this isn't how our controls add borders, we add them to the outside of the control, which leads to it adding to its height. Messing with a button's height between modes could also affect how it lays out its contents so it doesn't look consistent. I think this behavior is also seen in RNW because I've had to deal with a similar situation in a different app, not sure about other platforms. I think it was a technical limitation because of how XAML deals with borders (adds them to the outside)

A very hacky idea that may or may not work for various reasons is that we wrap the button content in two views. We have each one have a 1 px border and cap min height to 22 px for the inner view. Normally the inner border is the same color as the background unless the button is focused, in which case we would color the border the focus color. The outer border follows the coloring scheme for button borders as normal. This is based on how my old team used to fix similar issues in the past, where we would add a border regardless of whether the redlines wanted one and color it the same as the background color, so that when we did show a border, its height and contents wouldn't change.

If we were to fix it natively I would guess it would mean having to rework some layout logic? I'm not really knowledgeable on the native stuff though, so we may run into the same technical limitations as XAML, though.

Reply:
I think the ideal solution would be to fix native-side as 1. we would have to apply whatever workaround to any controls that have a similar treatment on focus and 2. any clients would have to do the same if they want similar behavior.

The alternative is that we go to design and tell them we have a limitation and we need to come up with something else. But it'd be nice to explore the native side solution first before doing that.

Reply:
Thanks for the info! It's really helpful to know. I'm not very familiar with layout and how it should work on the native side but I agree that it'd be ideal to see if we could fix this natively before applying a change here.